core/shims/common/src/main/scala/com/intel/raydp/shims/SparkShimLoader.scala

```diff
@@ -0,0 +1,61 @@
+package com.intel.raydp.shims
```

Comment:
can you please add apache license header for all new scale files?

Reply:
ok